filter/filter.go

```diff
@@ -151,9 +156,14 @@ func (f *Filter) Start() {
 		// check to see if relation (table name) is found in the lsit
 		found := false
 		for _, item := range f.tablelist {
-			if msg.Pr.Relation == item {
+			if f.regex == false && msg.Pr.Relation == item {
 				found = true
 				break
+			} else if f.regex == true {
+				if matched, _ := regexp.MatchString(item, msg.Pr.Relation); matched {
```

Comment:
For performances please compile the set of regexes outside of the outer for loop.

Reply:
Done.